src/test/resources/org/datadog/jmxfetch/default-jmx-metrics.yaml

```diff
@@ -0,0 +1,43 @@
+# This is a reduced set of 'default' metrics to make tests more predictable
```

Comment:
Same as above, let's keep compatibility with the metric names. You can use https://docs.datadoghq.com/integrations/java/#metrics

Reply:
This file is actually use only by tests, so it is not terribly important what specific metrics are inhere.

Reply:
Yes I know but I was hoping that the tests would catch the metric name changes to avoid regressions. Did they not ?

Reply:
I had to make some changes to tests to make build pass... So I guess they did to some extent.